gcb/publish/cloudbuild.yaml

```diff
@@ -70,12 +72,14 @@ substitutions:
   ## Optional/defaulted parameters
   _RELEASE_BUCKET: ""
   ## Options controlling the version of the release tooling used in the build.
-  _RELEASE_REPO_URL: https://github.com/cert-manager/release.git
-  _RELEASE_REPO_REF: "master"
+  _RELEASE_REPO_URL: https://github.com/wallrj/release.git
+  _RELEASE_REPO_REF: "3934-helm-github"
```

Comment:
I guess now is the time

Reply:
Done.

Reply:
It might be still pointing at the test repo at this point? 
https://github.com/wallrj/release/blob/3934-helm-github/gcb/publish/cloudbuild.yaml#L75-L76

Reply:
Oops. I fixed that in the test GCB file but not here.

Reply:
Done.